### PR TITLE
Homework 1-5

### PR DESCRIPTION
**Pull request by EnochRobokoff**

### CODE REVIEW

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 1]
```diff
@@ -9,7 +12,7 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array?.Length > 0) Array.Sort(array);
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs]: Hah! Please do not cheat, before you can use these functinos, you need to understand what happens under the hood, please implement it yourself.

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 2]
```diff
@@ -19,7 +22,7 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (array?.Length > 0) Array.Reverse(array);
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs]: Hah! Please do not cheat, before you can use these functinos, you need to understand what happens under the hood, please implement it yourself.

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 3]
```diff
@@ -39,7 +50,15 @@ public static int[] RemoveLast(int[] array)
         /// <returns>A new array with the first element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveFirst(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:67]: Reuse remove at

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 4]
```diff
@@ -29,7 +32,15 @@ public static void Reverse(int[] array)
         /// <returns>A new array with the last element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveLast(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:57]: Reuse remove at

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 5]
```diff
@@ -63,8 +103,17 @@ public static int[] RemoveAt(int[] array, int index)
         /// <returns>A new array with element added at a given index. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertFirst(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:107]: Reuse insert at

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 6]
```diff
@@ -75,8 +124,17 @@ public static int[] InsertFirst(int[] array, int number)
         /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertLast(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:118]: Reuse insert at

**Review by Almantask — CHANGES_REQUESTED** [review]

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 2 above)
Comment (EnochRobokoff) [reply Src/BootCamp.Chapter1/ArrayOperations.cs]: Whoops, busted! Okay, I'll try it the proper way.

**Review by Almantask — APPROVED** [review]
Looks good now 😄

**Pull request closed without merging**